src/renderer/shared/ui-kit/Select/Select.tsx

```diff
@@ -19,18 +19,18 @@ type ContextProps = {
 
 const Context = createContext<ContextProps>({});
 
-type ControlledSelectProps = {
+type ControlledSelectProps<T extends string> = {
   placeholder: string;
-  value: string;
-  onChange: (value: string) => void;
+  value: T | undefined | null;
```

Comment:
In order to avoid casting `ChainId` and other string types to `string`

Reply:
Why undefined and null at the same time? Let's get rid of undefined and and user will explicitly declare empty state